### PR TITLE
7.0 add meta files

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - VERSION="7.0" ODOO_REPO="odoo/odoo"
+  - VERSION="7.0" ODOO_REPO="OCA/OCB"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - travis_install_nightly ${VERSION}
```
Comment (lmignon) [inline .travis.yml]: ${VERSION} is no more required by travis_install_nigthly nor travirs_run_tests
- travis_install_nigthly
- travis_run_tests

Comment (eLBati) [inline .travis.yml]: > ${VERSION} is no more required by travis_install_nigthly nor 
> travirs_run_tests
> ...
> - travis_install_nigthly ...
> - travis_run_tests

Fixed. Thanks

**Comment by pedrobaeza** [comment]
:+1: for metafiles, but there is an error about a missing module, `product_customer_code`, that we will have to fix.

**Comment by lmignon** [comment]
:+1:  with the same remark as @pedrobaeza 
Thanks

**Pull request merged**